src/js/api/githubService.js

```diff
@@ -43,17 +42,25 @@ function loadPullRequestReactions(owner, repo, number) {
   return apiCall(url, { Accept: 'application/vnd.github.squirrel-girl-preview' });
 }
 
+function loadCommitStatus(owner, repo, sha) {
+  const url = `${config.apiBaseUrl}/repos/${owner}/${repo}/commits/${sha}/status`;
+  return apiCall(url);
+}
+
 export function getPullRequestDetails(owner, repo, number) {
-  return Promise.all([
-    loadPullRequest(owner, repo, number),
-    loadPullRequestComments(owner, repo, number),
-    loadPullRequestReactions(owner, repo, number)
-  ]).then(results => {
-    const [pullRequest, comments, reactions] = results;
-    return Object.assign(pullRequest.data, {
-      computedComments: comments.data,
-      computedReactions: reactions.data
-    });
+  return loadPullRequest(owner, repo, number).then(pullRequest => {
+    const { _links, head } = pullRequest.data;
+    return Promise.all([
+      loadPullRequestComments(_links.comments.href),
```

Comment:
Since I needed to run `loadCommitStatus` after `loadPullRequest` in order to get the commit SHA, I moved all of the functions that get further PR information into a nested Promise. This also allowed me to use the `_links.commits.href` instead of creating it manually.